src/v2/view-builder/views/email/ChallengeAuthenticatorEmailView.js

```diff
@@ -86,8 +88,16 @@ const Body = BaseAuthenticatorEmailForm.extend(
 
     postRender() {
       BaseAuthenticatorEmailForm.prototype.postRender.apply(this, arguments);
-
-      this.showCodeEntryField(false);
+      const useEmailMagicLink = this.options.appState.get('currentAuthenticatorEnrollment')?.
```

Comment:
create a new local function for `useEmailMagicLink()` that returns true and false.. to reuse it.